python/tidy/tidy.py

```diff
@@ -986,27 +1015,6 @@ def check_dep_license_errors(filenames, progress=True):
             if not ok_licensed:
                 yield (filename, 0, "dependency should contain a valid license.")
 
-
-class LintRunner(object):
-    def __init__(self, lint_path=None, only_changed_files=True,
-                 exclude_dirs=[], progress=True, stylo=False, no_wpt=False):
-        self.only_changed_files = only_changed_files
-        self.exclude_dirs = exclude_dirs
-        self.progress = progress
-        self.path = lint_path
-        self.stylo = stylo
-        self.no_wpt = no_wpt
-
-    def check(self, lint_cls):
-        lint = lint_cls(self.path, self.only_changed_files,
-                        self.exclude_dirs, self.progress,
-                        stylo=self.stylo, no_wpt=self.no_wpt)
-        for error in lint.run():
-            if type(error) is not tuple or (type(error) is tuple and len(error) != 3):
-                yield (self.path, 1, "errors should be a tuple of (path, line, reason)")
-                return
-            yield error
-
     def get_files(self, path, **kwargs):
```

Comment:
This method and the below `run` method are members of the `LintRunner` class and can be deleted as well.

Reply:
Nice catch! I've removed these.